tests/conftest.py

```diff
@@ -38,6 +38,8 @@
 os.environ["AIRFLOW__CORE__UNIT_TEST_MODE"] = "True"
 os.environ["AWS_DEFAULT_REGION"] = os.environ.get("AWS_DEFAULT_REGION") or "us-east-1"
 os.environ["CREDENTIALS_DIR"] = os.environ.get("CREDENTIALS_DIR") or "/files/airflow-breeze-config/keys"
+os.environ["AIRFLOW_ENABLE_AIP_44"] = os.environ.get("AIRFLOW_ENABLE_AIP_44") or "true"
```

Comment:
This is the default if not set when our tests are run. This way the user wil not have to set the variables in their tests when they run in from the IDE (cc: @ephraimbuddy @jedcunningham @eladkal @pierrejeambrun ). I found it highly annoying that I had to set it in main - but still in airflow they will be disabled by default.

In CI in branch 2.6 we will set the vars to be "false" by default, so the tests will be skipped there - even if the change here will be part of 2.6.